src/main.py

```diff
@@ -263,6 +263,9 @@ def __init__(self, aconfig):
 
 		self._aconfig = aconfig
 
+		if not self._aconfig['refresh_token']:
```

Comment:
Well, the exception crashes AlexaPi.

We can't go on. The setup (auth_web.py) adds refresh_token and we need it in order to obtain AVS token to communicate.